collect/collect.go

```diff
@@ -270,7 +270,7 @@ func (i *InMemCollector) checkAlloc() {
 		WithField("num_traces_sent", len(tracesSent)).
 		WithField("datasize_sent", totalDataSizeSent).
 		WithField("new_trace_count", i.cache.GetCacheCapacity()).
-		Logf("evicting large traces early due to memory overage")
+		Logf("Making some trace decisions early due to memory overrun.")
```

Comment:
A few lines up this is deemed an `error` severity message. Since it's not dropping data, does it make sense to downgrade it to `warning`?

Reply:
Good catch! I agree.

Reply:
Fixed in [c50a4ca](https://github.com/honeycombio/refinery/pull/1322/commits/c50a4ca093fa8937ec59ae810d34a29c151b1fc0)